### PR TITLE
Feature/sig 549 add priority

### PR DESCRIPTION
**Pull request by jasperswart**
added urgentie to wizard, admin list, filter and detail

### CODE REVIEW

File: src/signals/incident-management/containers/IncidentDetailPage/components/IncidentDetail/index.test.js [hunk 1]
```diff
@@ -25,17 +27,16 @@ describe('<IncidentDetail />', () => {
...
     expect(wrapper).toMatchSnapshot();
   });
 
   it('should render correctly without extra properties', () => {
     delete incident.extra_properties;
     const wrapper = shallow(
-      <IncidentDetail incident={incident} stadsdeelList={stadsdeelList} />
+      <IncidentDetail incident={incident} stadsdeelList={stadsdeelList} priorityList={priorityList} />
     );
-    expect(wrapper.find('tr').length).toEqual(14);
```
Comment (RobertLodder) [inline src/signals/incident-management/containers/IncidentDetailPage/components/IncidentDetail/index.test.js]: No replacement for this check?

File: src/signals/incident-management/containers/IncidentOverviewPage/components/Filter/index.js [hunk 2]
```diff
@@ -72,9 +74,11 @@ class Filter extends React.Component {
     );
   }
 }
+//                   <FieldControlWrapper render={SelectInput} name="incident_priority" display="Urgentie" control={this.filterForm.get('incident_priority')} values={priorityList} />
```
Comment (RobertLodder) [inline src/signals/incident-management/containers/IncidentOverviewPage/components/Filter/index.js]: Commented out code, please remove

File: src/signals/incident/containers/IncidentContainer/reducer.js [hunk 3]
```diff
@@ -13,44 +13,49 @@ import {
...
-    // subcategory: 'Overlast op het water - snel varen',
-    // category: 'Overlast op het water',
-    // description: 'snel boot',
-    // description: 'poep',
-    // phone: '020654321',
-    // email: 'a@b.com',
-    // extra_boten_snelheid_rondvaartboot: 'Ja',
-    // extra_boten_snelheid_rederij: 'Aemstelland',
-    // datetime: 'Nu',
+    location: {
```
Comment (RobertLodder) [inline src/signals/incident/containers/IncidentContainer/reducer.js]: This is not a correct initial state I believe

File: src/signals/incident/definitions/wizard-step-1-beschrijf.js [hunk 4]
```diff
@@ -43,6 +43,23 @@ export default {
         authenticated: true,
         render: FormComponents.SelectInput
       },
+      priority: {
+        meta: {
+          className: 'col-sm-12 col-md-6',
+          label: 'Wat is de prioriteit?',
```
Comment (RobertLodder) [inline src/signals/incident/definitions/wizard-step-1-beschrijf.js]: This should be 'urgentie'?

File: src/signals/incident/definitions/wizard-step-5-samenvatting.js [hunk 5]
```diff
@@ -19,6 +19,15 @@ export default {
         render: PreviewComponents.PlainText,
         optional: true
       },
+      priority: {
+        label: 'Prioriteit',
```
Comment (RobertLodder) [inline src/signals/incident/definitions/wizard-step-5-samenvatting.js]: This should be 'Urgentie'?

File: src/signals/incident-management/containers/IncidentPriorityContainer/components/Add/index.js [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+                    name="priority"
+                    display="Subcategorie"
+                    control={this.priorityForm.get('priority')}
+                    values={priorityList}
+                    multiple={false}
+                    emptyOptionText="Selecteer..."
+                  />
+
+                  <button className="action primary" type="submit" disabled={invalid || loading}>
+                    <span className="value">Subcategorie wijzigen</span>
```
Comment (RobertLodder) [inline src/signals/incident-management/containers/IncidentPriorityContainer/components/Add/index.js]: Copy/paste error?

File: src/signals/incident-management/containers/IncidentPriorityContainer/components/Add/index.js [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+        <div className="incident-priority-add__body">
+          <FieldGroup
+            control={this.priorityForm}
+            render={({ invalid }) => (
+              <form onSubmit={this.handleSubmit}>
+                <div>
+                  <FieldControlWrapper
+                    render={SelectInput}
+                    name="priority"
+                    display="Subcategorie"
```
Comment (RobertLodder) [inline src/signals/incident-management/containers/IncidentPriorityContainer/components/Add/index.js]: Copy/paste error?

File: src/signals/incident-management/containers/IncidentOverviewPage/components/List/index.js [hunk 8]
```diff
@@ -27,6 +27,7 @@ class List extends React.Component { // eslint-disable-line react/prefer-statele
                 <th className="">Subcategorie</th>
                 <th className="">Afdeling</th>
                 <th className="">Status</th>
+                <th className="">Urgentie</th>
```
Comment (RobertLodder) [inline src/signals/incident-management/containers/IncidentOverviewPage/components/List/index.js]: Even on a large monitor, the table does not fit correctly in the current design, I think we should fix that. Maybe by allowing the container/table to have a larger width?

File: src/signals/incident-management/containers/IncidentDetailPage/components/IncidentDetail/index.test.js (hunk 1 above)
Comment (jasperswart) [reply src/signals/incident-management/containers/IncidentDetailPage/components/IncidentDetail/index.test.js]: nee deze check is niet nodig want het aantal regels in de detail wordt (beter) bijgehouden in de snapshot. toch?

File: src/signals/incident/definitions/wizard-step-1-beschrijf.js (hunk 4 above)
Comment (jasperswart) [reply src/signals/incident/definitions/wizard-step-1-beschrijf.js]: sorry

File: src/signals/incident/containers/IncidentContainer/reducer.js (hunk 3 above)
Comment (jasperswart) [reply src/signals/incident/containers/IncidentContainer/reducer.js]: sorry

File: src/signals/incident/definitions/wizard-step-5-samenvatting.js (hunk 5 above)
Comment (jasperswart) [reply src/signals/incident/definitions/wizard-step-5-samenvatting.js]: gedaan

File: src/signals/incident-management/containers/IncidentDetailPage/components/IncidentDetail/index.test.js (hunk 1 above)
Comment (RobertLodder) [reply src/signals/incident-management/containers/IncidentDetailPage/components/IncidentDetail/index.test.js]: I like an explicit check, as missing it in the (large) snapshot code is not so hard when updating the same or another snapshot

File: src/signals/incident-management/containers/IncidentPriorityContainer/components/Add/index.js (hunk 6 above)
Comment (jasperswart) [reply src/signals/incident-management/containers/IncidentPriorityContainer/components/Add/index.js]: gefixt

File: src/signals/incident-management/containers/IncidentPriorityContainer/components/Add/index.js (hunk 7 above)
Comment (jasperswart) [reply src/signals/incident-management/containers/IncidentPriorityContainer/components/Add/index.js]: gefixt

File: src/signals/incident-management/containers/IncidentOverviewPage/components/List/index.js (hunk 8 above)
Comment (jasperswart) [reply src/signals/incident-management/containers/IncidentOverviewPage/components/List/index.js]: ja goed punt ik ga ernaar kijken

File: src/signals/incident/containers/IncidentContainer/reducer.js (hunk 3 above)
Comment (RobertLodder) [reply src/signals/incident/containers/IncidentContainer/reducer.js]: I'd prefer it if this test initial state is at least moved to a separate file

**Pull request merged**